Gems/ROS2/Code/Include/ROS2/Lidar/LidarRaycasterBus.h

```diff
@@ -173,6 +174,37 @@ namespace ROS2
             AZ_Assert(false, "This Lidar Implementation does not support Max range point addition configuration!");
         }
 
+        //! Enables and configures Point Cloud Publisher.
+        //! If not called no publishing is performed.
+        //! @param topicName Name of the ROS 2 topic the pointcloud is published on.
+        //! @param frameId Id of the ROS 2 frame of the sensor.
+        //! @param qoSPolicy QoS policy of published pointcloud messages.
+        virtual void ConfigurePointCloudPublisher(
```

Comment:
This API & documentation can be confusing since all sensors publish their data.
If this API is not called, contrary to what the second sentence claims, the publishing is still going to happen, just not in the raycaster (but in the component).

Keeping the API independent is nice but we still need to make it clear since most commonly raycaster will be used together with the component.

I think that adding justification (why?) would be good. 

"For some implementations, it might be beneficial to publish internally in the raycaster to avoid data copying. This API enables Raycaster implementations that also handle publishing and provides them with necessary publisher configuration"

Reply:
Thanks for pointing that out! Fixed.